src/main/java/org/opentripplanner/graph_builder/module/AddTransitModelEntitiesToGraph.java

```diff
@@ -205,7 +205,8 @@ private void createPathwayEdgesAndAddThemToGraph(Graph graph) {
                         pathway.getLength(),
                         pathway.getStairCount(),
                         pathway.getSlope(),
-                        pathway.isPathwayModeWheelchairAccessible()
+                        pathway.isPathwayModeWheelchairAccessible(),
+                        pathway.getId()
```

Comment:
Minor thing, but try to keep the constructor arguments in the order they are defined. The`id` is normally the first field in an entity class, but since the class inherit edge, put the id after the `toVertex` in the list.